packages/hooks/src/utils/index.ts

```diff
@@ -6,3 +6,6 @@ export const isString = (value: unknown): value is string => typeof value === 's
 export const isBoolean = (value: unknown): value is boolean => typeof value === 'boolean';
 export const isNumber = (value: unknown): value is number => typeof value === 'number';
 export const isUndef = (value: unknown): value is undefined => typeof value === 'undefined';
+
+export const isReactRef = (target: unknown): boolean =>
```

Comment:
```ts
```ts
export const isReactRef = <T extends TargetType>(
  target: any,
): target is MutableRefObject<TargetValue<T>> => {
  return target && target?.hasOwnProperty('current') && Object.keys(target).length === 1;
}
```
```